server/clickhouse/schema.sql

```diff
@@ -34,3 +34,39 @@ CREATE INDEX IF NOT EXISTS idx_tool_type ON http_requests_raw (tool_type) TYPE s
 CREATE INDEX IF NOT EXISTS idx_status_code ON http_requests_raw (status_code) TYPE set (100) GRANULARITY 4;
 CREATE INDEX IF NOT EXISTS idx_tool_urn_exact ON http_requests_raw (tool_urn) TYPE bloom_filter(0.01) GRANULARITY 4;
 CREATE INDEX IF NOT EXISTS idx_tool_urn_substring ON http_requests_raw (tool_urn) TYPE ngrambf_v1(4, 30720, 3, 0) GRANULARITY 4;
+
+CREATE TABLE IF NOT EXISTS tool_logs (
+    timestamp DateTime64(3, 'UTC') CODEC(Delta, ZSTD),
+    instance String CODEC(ZSTD),
+    level LowCardinality(String),
+    source LowCardinality(String),
+
+    raw_log String CODEC(ZSTD),
+    function_log Nullable(String) CODEC(ZSTD),
+
+    project_id UUID,
+    deployment_id UUID,
+    function_id UUID
+) ENGINE = MergeTree()
+PARTITION BY toYYYYMMDD(timestamp)
+ORDER BY (project_id, timestamp, instance)
+TTL timestamp + INTERVAL 30 DAY
+SETTINGS index_granularity = 8192
+COMMENT 'Stores logs from Gram function executions';
+
+ALTER TABLE tool_logs COMMENT COLUMN timestamp 'Timestamp at which log was generated (from the server)';
```

Comment:
Should these be ALTER TABLE in this file? Would prefer to have them as:

```sql
COMMENT ON COLUMN tool_logs.instance IS 'Name of the machine instance that generated the log (e.g. snowy-water-123)';
-- ...
```

... and then have atlas derive the ALTER TABLE statements in migration files. At the moment, they dont even appear in the migrations.